Automattic/wp-calypso
# Mock WordPress.com: Move to flags

### Changes

Follow up to https://vertexp2.wordpress.com/2024/06/17/mock-wordpress-com-in-30-seconds/#comment-2059

### Testing steps
1. Run `yarn`.
2. Run `yarn start-mock-wordpress-com`.
3. Calypso should serve at `https://wordpress.com:443`.


### Backwards compat
1. Run `bin/mock-wordpressdotcom`.
2. It should tell `Please use yarn start-mock-wordpress-com`
